### PR TITLE
feat: lines example

### PR DESCRIPTION
**Pull request by czizzy**

### CODE REVIEW

File: playground/lines.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
+/**
+ * @title Lines
+ * @category 2D
```
Comment (zhuxudong) [inline playground/lines.ts:3]: @category Toolkit

File: playground/lines.ts [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+]
+function change() {
+	if(lines.length) {
+		lines.forEach(line => {
+			line.cap = controls.cap;
+			line.join = Number(controls.join);
+			line.width = Number(controls.width);
+		})
+	}
+}
```
Comment (zhuxudong) [inline playground/lines.ts:47]: move  14~47 lines to "function showDebug()"

File: playground/lines.ts [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+
+// Create camera.
+const cameraEntity = rootEntity.createChild("Camera");
+const center = new Vector3(0, 0, 0);
+cameraEntity.transform.setPosition(center.x, center.y, 50);
+cameraEntity.transform.lookAt(center);
+const camera = cameraEntity.addComponent(Camera);
+camera.isOrthographic = true;
+engine.sceneManager.activeScene.background.solidColor = new Color(0.8, 0.8, 0.8, 1);
```
Comment (zhuxudong) [inline playground/lines.ts:63]: engine.sceneManager.activeScene.background.solidColor.set(0.8, 0.8, 0.8, 1);

**Review by zhuxudong — CHANGES_REQUESTED** [review]

File: playground/lines.ts (hunk 1 above)
Comment (czizzy) [reply playground/lines.ts]: fixed

File: playground/lines.ts (hunk 2 above)
Comment (czizzy) [reply playground/lines.ts]: fixed

File: playground/lines.ts (hunk 3 above)
Comment (czizzy) [reply playground/lines.ts]: fixed

**Pull request merged**